src/app/modules/knowledge-analytics/kc-statistics/events-table/events-table.component.ts

```diff
@@ -10,6 +10,7 @@ import { PageEvent } from '@angular/material/paginator';
   styleUrls: ['./events-table.component.scss'],
 })
 export class EventsTableComponent implements OnInit {
+  // TODO: Remove or rework component, it is currently unused.
```

Comment:
Has the ticket been created on ClickUp?

Reply:
No, but we should document this there instead of in code :)